README.md

```diff
@@ -93,8 +93,10 @@ Betaflight is forked from Cleanflight, so thanks goes to all those whom have con
 Origins for this fork (Thanks!):
 * **Alexinparis** (for MultiWii),
 * **timecop** (for Baseflight),
-* **Dominic Clifton** (for Cleanflight), and
-* **Sambas** (for the original STM32F4 port).
+* **Dominic Clifton** (for Cleanflight),
+* **Sambas** (for the original STM32F4 port), 
+* **Boris B.** (for Betaflight), and
+* **mikeller** (Rules betaflight with an iron first).
```

Comment:
This doesn't align with the purpose of this list, e.g. merits of each person listed